public/app/plugins/datasource/loki/types.ts

```diff
@@ -49,6 +49,7 @@ export interface LokiQuery extends DataQuery {
   /* @deprecated now use queryType */
   instant?: boolean;
   editorMode?: QueryEditorMode;
+  refId: string;
```

Comment:
Oh, gotcha. My editor was not picking it up from '@grafana/data': 

![imagen](https://user-images.githubusercontent.com/1069378/182622711-b18b3567-02a0-4920-82ca-550e29a4951f.png)

Do you use anything in particular to get those hints?

Reply:
Another example:

![imagen](https://user-images.githubusercontent.com/1069378/182623153-10f8ce54-09eb-4177-85e8-702c0e3421e0.png)

It will be super challenging to develop without having all that type context data.

Reply:
I don't have this problem. 😕 My editor correctly picks up on this. I would maybe try clicking on `@grafana/data` in https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/loki/datasource.ts#L36 to see if it helps editor figuring this out.

Reply:
Weird. I'll see if there's any setting that I can use.